horovod/torch/__init__.py

```diff
@@ -150,3 +199,45 @@ def broadcast_parameters(params, root_rank):
     # Wait for completion.
     for handle in handles:
         synchronize(handle)
+
+
+def save_model(filepath, model, optimizer=None, custom_state=None):
```

Comment:
I think we can nuke these, as there's nothing Horovod-specific.  Just doing `torch.save()` of dict would be almost as short.  WDYT?

Reply:
Makes sense.  Would you also then remove the `load_model` function as well?  It would break parity with the Keras API, but there's nothing Horovod specific there either.

Reply:
Right, these are too simple.  Keras has that function because it has magical optimizer wrappers.  As long as you can still do similar things in both frameworks, there's parity. :-)